### PR TITLE
Andrew goenner

### PR DESCRIPTION
**Pull request by drewgoenner**
Completed MVP, added line breaks and an onclick event button for stretch.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -40,3 +40,140 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const navLink = document.querySelectorAll('a');
```
Comment (thereactgirl) [inline js/index.js:44]: Awesome job creating a selector for the Nav!

File: js/index.js [hunk 2]
```diff
@@ -40,3 +40,140 @@ const siteContent = {
...
+navLink[2].setAttribute ('class', siteContent['nav']['nav-item-3']);
+navLink[2].text = 'Vision';
+navLink[2].style.color = 'green';
+navLink[3].setAttribute ('class', siteContent['nav']['nav-item-4']);
+navLink[3].text = 'Features';
+navLink[3].style.color = 'green';
+navLink[4].setAttribute ('class', siteContent['nav']['nav-item-5']);
+navLink[4].text = 'About';
+navLink[4].style.color = 'green';
+navLink[5].setAttribute ('class', siteContent['nav']['nav-item-6']);
```
Comment (thereactgirl) [inline js/index.js:60]: Here is an example of doing it the way we talked about in stand up using the forEach method. This code loops through the nodelist you have selected and assigns values from the siteContent object. 
```
nav.forEach(function(value, i){
  value.textContent=siteContent.nav[`nav-item-${i+1}`];
});
```

File: js/index.js [hunk 3]
```diff
@@ -40,3 +40,140 @@ const siteContent = {
...
+ctaH1.textContent = 'DOM \r \n';
+ctaH1.textContent += 'Is \r \n';
+ctaH1.textContent += 'Awesome';
+
+const buttonText = document.querySelector('button');
+buttonText.textContent = siteContent['cta']['button'];
+const bigButton = document.querySelector('button');
+bigButton.addEventListener('click', (event) => {
+  const brokeCtaH1 = document.querySelector('h1');
+  brokeCtaH1.setAttribute('style', 'white-space: pre;');
```
Comment (rleslie1015) [inline js/index.js]: Very cool finding this CSS property for handling white space!

File: js/index.js [hunk 4]
```diff
@@ -40,3 +40,140 @@ const siteContent = {
...
+navLink[5].style.color = 'green';
+
+
+
+const ctaH1 = document.querySelector('h1');
+ctaH1.setAttribute('style', 'white-space: pre;');
+ctaH1.textContent = 'DOM \r \n';
+ctaH1.textContent += 'Is \r \n';
+ctaH1.textContent += 'Awesome';
+
```
Comment (thereactgirl) [inline js/index.js:71]: Very cool finding this CSS property for handling white space!

File: js/index.js [hunk 5]
```diff
@@ -40,3 +40,140 @@ const siteContent = {
...
+  navLink[3].style.color = 'green';
+  navLink[4].setAttribute ('class', siteContent['nav']['nav-item-5']);
+  navLink[4].text = 'DOOM';
+  navLink[4].style.color = 'green';
+  navLink[5].setAttribute ('class', siteContent['nav']['nav-item-6']);
+  navLink[5].textContent = "DOOM";
+  navLink[5].style.color = 'green';
+  alert('You broke the p...DOOM DOOM DOOM DOOM DOOM')
+})
+
```
Comment (thereactgirl) [inline js/index.js:102]: This is an insanely long function!! But I love the creativity and humor here!

**Review by thereactgirl — COMMENTED** [review]
Great job completing the project and stretch!!

**Pull request merged**